site/src/atomics/display.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Display Atomics
+description: Display related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Display Atomics
+
+The following displays are covered: `block`, `flex`, `inline`, `inline-block`, `inline-flex`, `grid`, and `none`.
+
+The following breakpoints are supported for each display value:
+
+```Text
+.display-{value}-tablet
+.display-{value}-desktop
+.display-{value}-widescreen
+```
+
+## Example
+
+```html
+<div class="border">
+	<button class="button display-block border">Foo</button>
+	<button class="button display-block border">Bar</button>
```

Comment:
\<button class="display-block">Foo<\/button>
\<button class="display-block-tablet">Foo<\/button>

?

Reply:
What about something like this?

```
<div class="display-block border">
	<div>Foo</div>
	<div>Foo</div>
</div>
<div class="display-flex-tablet border">
	<div>Bar</div>
	<div>Bar</div>
</div>
<div class="display-inline-desktop border">
	<div>Baz</div>
	<div>Baz</div>
</div>
<div class="display-inline-flex-widescreen border">
	<div>Foo</div>
	<div>Foo</div>
</div>
```